autosklearn/automl.py

```diff
@@ -716,7 +732,9 @@ def fit(
         # If no dask client was provided, we create one, so that we can
         # start a ensemble process in parallel to smbo optimize
         if self._dask_client is None and (
-            self._ensemble_size > 0 or self._n_jobs is not None and self._n_jobs > 1
+            self._ensemble_class is not None
```

Comment:
So `_ensemble_class = None` is the new indicator for no ensembling now I guess?

Reply:
Correct.